lld/test/ELF/aarch64-feature-gcs.s

```diff
@@ -0,0 +1,128 @@
+# REQUIRES: aarch64
+# RUN: rm -rf %t && split-file %s %t && cd %t
+# RUN: llvm-mc -filetype=obj -triple=aarch64-linux-gnu func1.s -o func1.o
+# RUN: llvm-mc -filetype=obj -triple=aarch64-linux-gnu func2.s -o func2.o
+# RUN: llvm-mc -filetype=obj -triple=aarch64-linux-gnu func2-gcs.s -o func2-gcs.o
+# RUN: llvm-mc -filetype=obj -triple=aarch64-linux-gnu func3.s -o func3.o
+# RUN: llvm-mc -filetype=obj -triple=aarch64-linux-gnu func3-gcs.s -o func3-gcs.o
+
+## GCS should be enabled when it's enabled in all inputs or when it's forced on.
+
+# RUN: ld.lld func1.o func2-gcs.o func3-gcs.o --shared -o gcs.exe
```

Comment:
We typically use `*.so` files for `-shared`. Executables omit the suffix.

Reply:
The ``--shared`` on this line (and others) that doesn't generate a .so was a mistake, I've removed it.